packages/www/pages/app/stream/[id].tsx

```diff
@@ -305,7 +305,7 @@ const ID = () => {
   const playbackId = (stream || {}).playbackId || "";
   const domain = isStaging() ? "monster" : "com";
   const globalIngestUrl = `rtmp://rtmp.livepeer.${domain}/live`;
-  const globalPlaybackUrl = `https://cdn.livepeer.${domain}/hls/${playbackId}/index.m3u8`;
+  const globalPlaybackUrl = `https://livepeercdn.${domain}/hls/${playbackId}/index.m3u8`;
```

Comment:
Either way, WDYT of adding a tooltip in these URLs explaining that we changed our playback domains, but old ones will continue working? If we aren't gonna have any docs or posts about it, I think it's kind of a must.

It's really easy to do so with the built-in `Tooltip` from our design system. e.g. https://github.com/livepeer/livepeer-com/blob/0442d063e7382f8e7af6c5497ac1bd9f5a638296/packages/www/components/Dashboard/MultistreamTargetsTable/SaveTargetDialog.tsx#L325